pkgs/os-specific/linux/device-tree/raspberrypi.nix

```diff
@@ -11,7 +11,7 @@ stdenvNoCC.mkDerivation {
 
     cp ${raspberrypifw}/share/raspberrypi/boot/bcm*.dtb .
 
-    cp bcm2708-rpi-0-w.dtb bcm2835-rpi-zero-w.dtb
+    cp bcm2708-rpi-zero-w.dtb bcm2835-rpi-zero-w.dtb
```

Comment:
Here's where the change comes from:

 * https://github.com/raspberrypi/linux/commit/6f91b5dbfdb62a434571a73f2dc15181963e3bea#diff-13c461528c1bfe12f245cebb28715f16

I don't know if we want this to handle both possibilities of an older downstream kernel with the older name for the dtb or not.

Reply:
Intesting. Here the dtb's are grabbed from `raspberrypifw` however, which just consists of binary blobs. Makes me question if that's the right approach, but for now, the firmware package has only one version in the repo, and it's quite unlikely anyone will override it with an older version, imo.

Reply:
Yeah, that was my thought too. I wasn't sure how likely a user would rely on an older kernel revision, while using a newer nixpkgs. Unlikely I guess, and at that point, they may as well patch this package in their overlay.